app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    parts = cp.split(" ")
+    if len(parts) != 3 or parts[0] != "cp":
+        raise Exception
+    source = parts[1]
+    destination = parts[2]
+    if source == destination:
+        return
```

Comment:
Simplify and combine two conditions into one.
if source == destination or command != "cp":
    return